<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>imodvDepthcueForm</class>
  <widget class="QWidget" name="imodvDepthcueForm" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>190</width>
        <height>207</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>3dmodv Depth Cue</string>
    </property>
    <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QCheckBox" name="enableBox" >
          <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
          </property>
          <property name="text" >
            <string>Depth Cue</string>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Turn depth cueing with fog on or off</string>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QSlider" name="startSlider" >
              <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
              </property>
              <property name="maximum" >
                <number>100</number>
              </property>
              <property name="pageStep" >
                <number>1</number>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Set relative depth where fog starts</string>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <property name="spacing" >
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="TextLabel3" >
                  <property name="text" >
                    <string>Starting depth</string>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignTop|Qt::AlignLeft</set>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="startTextLabel" >
                  <property name="text" >
                    <string>0</string>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignTop|Qt::AlignRight</set>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QSlider" name="endSlider" >
              <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
              </property>
              <property name="maximum" >
                <number>100</number>
              </property>
              <property name="pageStep" >
                <number>1</number>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Set relative depth where fog reaches maximum</string>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <property name="spacing" >
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="TextLabel3_2" >
                  <property name="text" >
                    <string>Ending depth</string>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignTop|Qt::AlignLeft</set>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="endTextLabel" >
                  <property name="text" >
                    <string>0</string>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignTop|Qt::AlignRight</set>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="Line1" >
          <property name="frameShape" >
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="doneButton" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>70</width>
                  <height>32767</height>
                </size>
              </property>
              <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
              </property>
              <property name="text" >
                <string>Done</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Close dialog box</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="helpButton" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>70</width>
                  <height>32767</height>
                </size>
              </property>
              <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
              </property>
              <property name="text" >
                <string>Help</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Open help window</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <connections>
    <connection>
      <sender>enableBox</sender>
      <signal>toggled(bool)</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>depthcueToggled(bool)</slot>
    </connection>
    <connection>
      <sender>endSlider</sender>
      <signal>sliderPressed()</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>endPressed()</slot>
    </connection>
    <connection>
      <sender>endSlider</sender>
      <signal>valueChanged(int)</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>endChanged(int)</slot>
    </connection>
    <connection>
      <sender>endSlider</sender>
      <signal>valueChanged(int)</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>displayEndLabel(int)</slot>
    </connection>
    <connection>
      <sender>endSlider</sender>
      <signal>sliderReleased()</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>endReleased()</slot>
    </connection>
    <connection>
      <sender>startSlider</sender>
      <signal>sliderPressed()</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>startPressed()</slot>
    </connection>
    <connection>
      <sender>startSlider</sender>
      <signal>valueChanged(int)</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>startChanged(int)</slot>
    </connection>
    <connection>
      <sender>startSlider</sender>
      <signal>sliderReleased()</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>startReleased()</slot>
    </connection>
    <connection>
      <sender>startSlider</sender>
      <signal>valueChanged(int)</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>displayStartLabel(int)</slot>
    </connection>
    <connection>
      <sender>doneButton</sender>
      <signal>clicked()</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>donePressed()</slot>
    </connection>
    <connection>
      <sender>helpButton</sender>
      <signal>clicked()</signal>
      <receiver>imodvDepthcueForm</receiver>
      <slot>helpPressed()</slot>
    </connection>
  </connections>
</ui>
